Tests/FuzzilliTests/LifterTest.swift

```diff
@@ -535,6 +535,41 @@ class LifterTests: XCTestCase {
 
         XCTAssertEqual(lifted_program,expected_program)
     }
+
+    func testAssignmentOperationLifting(){
+        let fuzzer = makeMockFuzzer()
+        let b = fuzzer.makeBuilder()
+
+        let v0 = b.loadInt(1337)
+        let v1 = b.loadFloat(13.37)
+        b.reassign(v0, to: v1)
```

Comment:
Maybe drop this `reassign` after fixing the `VariableAnalyzer` (see main comment)